test/OpenTelemetry.Exporter.Geneva.Tests/Testing.md

```diff
@@ -0,0 +1,188 @@
+# Manual testing steps: Reading events from user_events on Linux
+
+`UnixUserEventsDataTransport` class can write events to user_events on .NET 8
+and later on versions of Linux with user_events available in the kernel (6.6+).
+
+However, the agents in GitHub Action are not running on hosts with compatible kernel.
+Reading these events is a bit tricky because it needs elevation. Thus this
+documents records some manual steps to run tests that can validate the
+user_events feature works.
+
+## Testing
+
+### Prerequisites
+
+* Perf Tool: Install the [perf](https://perf.wiki.kernel.org/index.php/Main_Page)
+  tool to capture the user_events.
+* Decode-Perf Tool: Install the [decode-perf](https://github.com/microsoft/LinuxTracepoints/tree/main/libeventheader-decode-cpp/tools)
+  tool to decode the user_events.
+
+### Steps for testing UnixUserEventsDataTransport
+
+To capture the user_events, the perf tool has to be run while `dotnet test` is
+running. The most simple way is to do in two terminals.
+
+#### Terminal 1
+
+First, remove `(Skip = "This would fail on Ubuntu. Skipping for now. See
```

Comment:
In Terminal 2, you have a numbered list.
Here, you have paragraph form ("First").

I would use a numbered list here too for consistency.

Reply:
Updated